docs/articles/documentation/test-api/test-code-structure.md

```diff
@@ -572,3 +573,58 @@ test('Fixture2Test3', () => {});
 
 // Only tests in Fixture1 and the Fixture2Test2 test are run
 ```
+
+## Inject Scripts into Tested Pages
+
+TestCafe allows you to [inject custom scripts](../using-testcafe/common-concepts/inject-scripts-into-tested-pages.md) into pages visited during the tests. You can add scripts that mock browser API or provide helper functions.
+
+Use the `fixture.clientScripts` and `test.clientScripts` methods to add scripts to pages visited during a particular test or fixture.
+
+```text
+fixture.clientScripts( script[, script2[, ...[, scriptN]]] )
+```
+
+```text
+test.clientScripts( script[, script2[, ...[, scriptN]]] )
+```
+
+Parameter | Type     | Description
+--------- | -------- | ---------------------------------------------------------------------------
+`script`, `script2`, `scriptN` | String &#124; Object &#124; Array | Scripts to inject into the tested pages. See [Provide Scripts to Inject](../using-testcafe/common-concepts/inject-scripts-into-tested-pages.md#provide-scripts-to-inject) to learn how to specify them.
```

Comment:
See [Provide Scripts to Inject](../using-testcafe/common-concepts/inject-scripts-into-tested-pages.md#provide-scripts-to-inject) for information on how to specify scripts.